src/app/components/annotations/pages/search/search.component.html

```diff
@@ -0,0 +1,72 @@
+<h1>Annotation Search</h1>
+
+<div>
+  <baw-annotation-search-form
+    [searchParameters]="searchParameters"
+    (searchParametersChange)="updateSearchParameters($event)"
+    [project]="project"
+    [region]="region"
+    [site]="site"
+  ></baw-annotation-search-form>
+
+  <div class="mt-2">
+    <!--
+      TODO: Once the audio events download route supports url encoded filters
+      we should uncomment this buttons functionality
```

Comment:
file an api issue and link back

Reply:
Blocked by https://github.com/QutEcoacoustics/baw-server/issues/688